Update guide_coturn.rst (#712)
Uberspace/lab: source/guide_coturn.rst

Open Firewall Ports
-------------------

> Coturn needs at least 2 open ports, plus some additional sucessive ports as port range for UDP connections.
> An unberspace can open a maximum of 20 ports.
> So lets open 5 ports for now: two for base functions and three for UDP connections.

Reviewer: A couple of typos.

```suggestion
Coturn needs at least 2 open ports, plus some additional successive ports as port range for UDP connections.
An uberspace can open a maximum of 20 ports.
So let's open 5 ports for now: two for base functions and three for UDP connections.
```

Create a new config file at ``$HOME/etc/coturn/turnserver.conf``
Replace values in brackets ``<value>`` with your values.

> Note: ``total-quota`` is the limit for all simultanious udp connections. In this example we only opened three ports for UDP, so this limit is 3. If you open more ports for UDP connections, change this setting accordingly.

Reviewer: Please make this an actual `.. note::` box. You can copy the syntax from the "For this guide you should be familiar with (...)" block at the start of your guide.

 listening-ip=0.0.0.0
 relay-ip=::
 relay-ip=0.0.0.0

>  external-ip=<IP_OF_YOUR_UBERSPACE>

Reviewer: which IP address? the external one, e.g. `185.26.156.62` for `tuttle.uberspace.de`? Please add a comment or hint somewhere on how to get that IP.

 min-port=<port-3>
 max-port=<port-5>
 fingerprint
 use-auth-secret
 static-auth-secret=<YOUR_SUPER_LONG_SUPER_SECRET_STATIC_PASSPHRASE>

>  realm=<isabell>.uber.space

Reviewer: This should be `<username>.uber.space`. Please also add a note block above about the needed replacement so users know that they need to do something.

```suggestion
 realm=<username>.uber.space
```

 total-quota=3
 bps-capacity=0
 stale-nonce

>  cert=/home/isabell/etc/certificates/<isabell>.uber.space.crt
>  pkey=/home/isabell/etc/certificates/<isabell>.uber.space.key
>  cipher-list="ECDH+AESGCM:DH+AESGCM:ECDH+AES256:DH+AES256:ECDH+AES128:DH+AES:ECDH+3DES:DH+3DES:RSA+AES:RSA+3DES:!ADH:!AECDH:!MD5"
>  dh-file=/home/<isabell>/etc/coturn/dhparam-2066.pem
>  log-file=/home/<isabell>/logs/coturn.log

Reviewer: ```suggestion
 cert=/home/isabell/etc/certificates/<username>.uber.space.crt
 pkey=/home/isabell/etc/certificates/<username>.uber.space.key
 cipher-list="ECDH+AESGCM:DH+AESGCM:ECDH+AES256:DH+AES256:ECDH+AES128:DH+AES:ECDH+3DES:DH+3DES:RSA+AES:RSA+3DES:!ADH:!AECDH:!MD5"
 dh-file=/home/<username>/etc/coturn/dhparam-2066.pem
 log-file=/home/<username>/logs/coturn.log
```
`<username>` again.
